fetch.bs

```diff
@@ -5158,7 +5262,8 @@ these steps:
        <var>response</var>'s <a for=response>URL list</a> has more than one item.
       </ul>
 
-      <p>then return a <a>network error</a>.
+      <p>then return the result of running <a>HTTPS upgrade fallback</a> given <var>request</var>
```

Comment:
Should we only fallback in the "error" case above? (for the rest it doesn't seem like a fallback would help)

Reply:
Done